### PR TITLE
Add BLM solidarity statement

### PR DESCRIPTION
**Pull request by isaacdurazo**
@vladikoff Here's my PR per discussed in issue #218. I would appreciate a code review on this 😃

# This is how the banner looks in every page
Screen Shot 2020-07-08 at 12 14 28 PM

# Here's a dedicated page for the Black Lives Matter Statement
Screen Shot 2020-07-08 at 12 14 39 PM

### CODE REVIEW

File: src/tmpl/blm.pug [hunk 1]
```diff
@@ -0,0 +1,47 @@
+doctype html
```
Comment (vladikoff) [inline src/tmpl/blm.pug:1]: @isaacdurazo Could you use the existing layout? Like this one for example https://github.com/gruntjs/gruntjs.com/blob/b89770f12c5c405cb46f2dc21491615fa26f6cc7/src/tmpl/plugins.pug ?

File: Gruntfile.js [hunk 2]
```diff
@@ -182,6 +182,19 @@ module.exports = function(grunt) {
...
+
+    pug: {
+      compile: {
+        options: {
+          data: {
+            debug: false
+          }
+        },
+        files: {
+          'build/blm.html': ['src/tmpl/blm.pug']
```
Comment (vladikoff) [inline Gruntfile.js:195]: There's already a task that picks up pages from templates. pages like plugins.html/pug and others. so you probably do not need this step or the extra plugin.

File: server.js [hunk 3]
```diff
@@ -113,6 +113,11 @@ app.get('/plugins*', function (req, res) {
   res.sendFile('build/plugins.html', {root: __dirname});
 });
 
+// blm route
+app.get('/blm*', function (req, res) {
```
Comment (vladikoff) [inline server.js:117]: the * route handler below should catch the blm.html, so you probably won't need this.

File: src/tmpl/blm.pug (hunk 1 above)
Comment (isaacdurazo) [reply src/tmpl/blm.pug:1]: @vladikoff I actually tried to do that, but the problem I was having is that 1) the banner at the top was getting included and 2) I needed to modify the nav, specifically the logo as you can see in the screenshot. 

It's my first time using Pug and tried to find a way to do it with a conditional statement or something like that, but couldn't find a good solution. Do you have a suggestion for a workaround?

File: Gruntfile.js (hunk 2 above)
Comment (isaacdurazo) [reply Gruntfile.js:195]: This is odd, that's exactly what I thought, but for some reason the file wasn't being built so I added that plugin and task.

**Comment by vladikoff** [comment]
deploying!

**Comment by vladikoff** [comment]
Thanks @isaacdurazo !!!

**Comment by isaacdurazo** [comment]
Thank you so much for the help @vladikoff 🙌

**Pull request merged**